src/openrct2-ui/scripting/CustomWindow.cpp

```diff
@@ -66,8 +66,8 @@ namespace OpenRCT2::Ui::Windows
         TextAlignment TextAlign{};
         colour_t Colour{};
         uint8_t Percentage{};      // progress bar value
-        uint8_t LowerBlinkBound{}; // progress bar blink when below
-        uint8_t UpperBlinkBound{}; // progress bar blink when above
+        uint8_t LowerBlinkBound{}; // progress bar will blink when above the lower bound
+        uint8_t UpperBlinkBound{}; // and below upper bound
```

Comment:
I agree that it's a bit unintuitive. I'll make a note in the .d.ts that the bar will blink when the percentage is between the upper and lower bound values, which I think will clear it up a bit. And I think this confirms keeping a strictly numerical value instead of using `null`.